src/proto/message_translation/power_frame_msg.hpp

```diff
@@ -146,7 +149,10 @@ TbotsProto_PowerPulseControl inline createNanoPbPowerPulseControl(
         case TbotsProto::PowerControl::ChickerControl::kChipDistanceMeters:
             nanopb_control.chicker.which_chicker_command =
                 TbotsProto_PowerPulseControl_ChickerControl_chip_pulse_width_tag;
-            nanopb_control.chicker.chicker_command.chip_pulse_width = chip_pulse_width;
+            nanopb_control.chicker.chicker_command.chip_pulse_width =
```

Comment:
can you limit the max chip_pulse_width to a safe number (ask elec)

Reply:
It is limited, see:
https://github.com/UBC-Thunderbots/Software/pull/3265/files#diff-45ff27e52591072cc56ef258c891210d55e70f0cddfa1c5415691c0b7684f1f6R136-R137

This value can be tuned further though.

Reply:
Actually I thought of a better way to limit it by moving the min into the switch case directly, Will fix

Reply:
fixed